.travis.yml

```diff
@@ -8,11 +8,15 @@ env:
 
 install:
   - pip install -q -r requirements.txt
-  - git clone https://github.com/tomopy/tomopy --branch master --single-branch ~/tomopy
-  - pushd ~/tomopy
+  - pushd ~
+  - wget https://github.com/tomopy/tomopy/archive/1.2.1.tar.gz -O tomopy.tar.gz
+  - tar -xvf tomopy.tar.gz
+  - pushd ~/tomopy-1.2.1/
```

Comment:
remove the line of python build.py ?

Reply:
@licode, it's the expected behavior in this case. The old installation method, which is used in v1.2.1 (released back in October 2018) relied on `python build.py` to build the shared libraries. However, in November they updated the building approach, and now everything is done with `pip install .` (see https://github.com/tomopy/tomopy/commit/21632927c3c78a5e3a4e3e01f67e4ef5b04a3edd).

I tried this new approach in #97, and it worked fine. Unfortunately, I didn't mention to keep it unmerged until after there is a newer tag of tomopy, so that we can use a tagged version as we did here in this PR. Anyway, this PR can be rebased on master with resolution of the conflicts to incorporate the changes from #97. For now we can continue using tomopy from the master branch.